tests/ltp/ltp-batch1/ltp_disabled_tests.txt

```diff
@@ -261,10 +261,10 @@
 #/ltp/testcases/kernel/syscalls/fstatat/fstatat01
 #/ltp/testcases/kernel/syscalls/fstatfs/fstatfs01
 #/ltp/testcases/kernel/syscalls/fstatfs/fstatfs02
-#/ltp/testcases/kernel/syscalls/fsync/fsync01
-#/ltp/testcases/kernel/syscalls/fsync/fsync02
-#/ltp/testcases/kernel/syscalls/fsync/fsync03
-#/ltp/testcases/kernel/syscalls/fsync/fsync04
+/ltp/testcases/kernel/syscalls/fsync/fsync01
+/ltp/testcases/kernel/syscalls/fsync/fsync02
```

Comment:
fsync02 and fsync03 is not even patched. They are original LTP tests. Why should we disable them? Do you mean they are not supported by sgx-lkl?

https://github.com/linux-test-project/ltp/blob/master/testcases/kernel/syscalls/fsync/fsync02.c
https://github.com/linux-test-project/ltp/blob/master/testcases/kernel/syscalls/fsync/fsync03.c

Reply:
I suggest that we look at all of the tests together and first decide how they test fsync and if this makes sense for SGX-LKL.